code/src/Teletubbies.css

```diff
@@ -0,0 +1,24 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap");
+
+.teletubbies-wrapper{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
```

Comment:
One suggestion here since I see that you style all your question-wrappers the same, would be to just have one class-name for example question-wrapper, that you then only have to style once.

Reply:
I saw now that you also have a section named question-wrapper above the div. But maybe you could do without the div and only style the section (then the button would also be centered of course)? Just a suggestion :)

Reply:
I'll take that with me for the future! :D Thanks